command/connect/proxy/flag_upstreams.go

```diff
@@ -15,6 +15,10 @@ import (
 //
 // The syntax of the value is "name:addr" where addr can be "port" or
 // "host:port". Examples: "db:8181", "db:127.0.0.10:8282", etc.
+//
+// This would need to be extended for Unix Domain Sockets; how does this get handled
```

Comment:
This code is for the built-in proxy, so we could just add warnings about how it doesn't understand UDS yet to those website sections.